src/Bridge/Doctrine/Orm/Filter/AbstractFilter.php

```diff
@@ -116,7 +116,7 @@ protected function isPropertyEnabled(string $property): bool
      */
     protected function isPropertyMapped(string $property, string $resourceClass, bool $allowAssociation = false): bool
     {
-        if ($this->isPropertyNested($property)) {
+        if ($this->isPropertyNested($property) && !$this->isPropertyNestedEmbedded($property, $resourceClass)) {
```

Comment:
@teohhanhui @soyuka , what about extractProperties method in the class AbstracFilter ? The property $needsFixing to true. Otherwise the embed field embed_b.field is transformed to embed_b_field and generates an error in the request